Misc/NEWS.d/next/Core and Builtins/2024-04-30-14-03-09.gh-issue-117151.yt2H8c.rst

```diff
@@ -0,0 +1,3 @@
+Increase ``io.DEFAULT_BUFFER_SIZE`` from 4k/8k to 128k. Adapt :func:`open` on Linux to use
```

Comment:
I think that this should be rather in the "Library" section than in "Core and Builtins".

Reply:
This was set here by the news tool automatically. I think "Core and Builtins" makes more sense for the I/O buffering code to be considered core and builtins, though I can move if you prefer?

Reply:
While `open()` is exposed as a builtin, it is defined in the `io` module. This is not a part of the core interpreter.